spec/controllers/canvas_assignment_overrides_controller_spec.rb

```diff
@@ -0,0 +1,99 @@
+require 'rails_helper'
+
+RSpec.describe CanvasAssignmentOverridesController, type: :controller do
+  render_views
+
+  context "with normal signin" do
+    let(:user) { create :admin_user }
+    let(:course) { create :course }
+    let(:section) { create :section, course: course }
+    let(:from_course) { create :course }
+    let(:from_section) { create :section, course: from_course }
```

Comment:
nit: can you do the following to make it clear the intention that this code runs only on cohort schedule sections?

```suggestion
    let(:from_section) { create :cohort_schedule_section, course: from_course }
```

Reply:
it's the only intended use case for after 2022, based on your proposal that we only put due dates in cohort schedules moving forward. as far as i'm aware, we only have due dates in regular sections right now, so we won't be able to do that for a year

Reply:
talked about this in working session and realized the controller does limit by cohort schedule, and we should have cohort schedule due dates for even old courses since that's where they're set up initially before being copied to individual cohort sections. updated specs to match